test_app.rb

```diff
@@ -38,14 +38,31 @@ def resources_pod(package_root, current_dir = package_root)
   podspec_path = File.join(current_dir, 'ReactTestApp-Resources.podspec.json')
   File.write(podspec_path, spec.to_json())
   at_exit { File.delete(podspec_path) }
-  Pathname.new(current_dir).relative_path_from(package_root).to_s()
+  Pathname.new(current_dir).relative_path_from(project_root).to_s()
```

Comment:
Just an FYI, but idiomatic Ruby is to omit parenthesis, when not passing arguments at least:

```suggestion
  Pathname.new(current_dir).relative_path_from(project_root).to_s
```

Reply:
Thanks, I really appreciate these types of comments. Do you know of a good linter we can use?

Reply:
👍 Rubocop is the linter of choice.